lib/fetch/sanitize.js

```diff
@@ -164,22 +168,46 @@ export function sanitizeBadger(json) {
     lastName: name && name.split(' ').slice(-1).join(' '),
     order: get('badger.order'),
     jobTitle: get('badger.job-title'),
-    primaryImageUrl: primaryImageUrl && primaryImageUrl.replace(/^http:/, 'https:'),
-    secondaryImageUrl: secondaryImageUrl && secondaryImageUrl.replace(/^http:/, 'https:'),
+    primaryImageUrl: primaryImageUrl &&
```

Comment:
Shouldn't be. I think this only changed because Alex has prettier turned on which auto formatted this code. This PR doesn't touch badgers